service-base/src/main/java/org/eclipse/hono/service/management/tenant/Adapter.java

```diff
@@ -95,4 +111,20 @@ public Adapter setExtensions(final Map<String, Object> extensions) {
     public Map<String, Object> getExtensions() {
         return this.extensions;
     }
+
+    /**
+     * Add a new extension entry to the Adapter.
```

Comment:
IMHO this should be changed to
> Adds or replaces an extension property to this adapter.

to also reflect the *put* characteristics ...